mysql/datadog_checks/mysql/statements.py

```diff
@@ -128,6 +135,12 @@ def collect_per_statement_metrics(self):
 
         rows = self._collect_per_statement_metrics()
         if not rows:
+            self._check.histogram(
```

Comment:
is there something different about this that we requires we keep it in here?

Reply:
oh whoops - no, just oversight.